content/verein.de.md

```diff
@@ -12,7 +12,7 @@ Einen Überblick über die Entwicklung der SwingConnection seit dem Jahr 2000 k
 Du hast in den Basicskursen Swingluft geschnuppert und willst mehr? Wir freuen uns immer über Zuwachs. Als Mitglied bei der SwingConnection Leipzig kannst du dich aktiv am Vereinsleben beteiligen. Außerdem kannst du das ganze Jahr über kostenlos an allen regulären Tanzkursen teilnehmen und bekommst ermäßigten Zugang zu weiteren Veranstaltungen, die die SCL organisiert. Falls du Fragen zur Mitgliedschaft hast, sprich uns gerne bei unseren Veranstaltungen an oder schreibe uns eine [E-Mail](info@swingconnection-leipzig.com).
 
 #### Jahresgebühr
-Aktuell beträgt die Jahresgebühr für eine Mitgliedschaft 180 € bzw. 144 € (ermäßigt). Die ermäßigte Gebühr gilt für Inhaber von Schülerausweis, Studentenausweis, Leipzig-Pass oder Schwerbehindertenausweis. Außerdem wird einmalig eine Aufnahmegebühr in Höhe von 20 € fällig.
+Aktuell beträgt die Jahresgebühr für eine Mitgliedschaft 190 € bzw. 154 € (ermäßigt). Die ermäßigte Gebühr gilt für Inhaber von Schülerausweis, Studentenausweis, Leipzig-Pass oder Schwerbehindertenausweis. Außerdem wird einmalig eine Aufnahmegebühr in Höhe von 30 € fällig.
```

Comment:
```suggestion
Aktuell beträgt die Jahresgebühr für eine Mitgliedschaft 230 € bzw. 194 € (ermäßigt). Die ermäßigte Gebühr gilt für Inhaber von Schülerausweis, Studentenausweis, Leipzig-Pass oder Schwerbehindertenausweis. Außerdem wird einmalig eine Aufnahmegebühr in Höhe von 30 € fällig.
```
Du hast hier nur den aktuellen Spartenbeitrag hier genommen. Es gibt aber noch den Grundbeitrag von 40 €. Das steht etwas komisch in der Beitragsordnung drin.

Reply:
Achso, alles klar.